lib/services/project-service.ts

```diff
@@ -80,7 +80,7 @@ export class ProjectService implements IProjectService {
 	public isValidNativeScriptProject(pathToProject?: string): boolean {
 		try {
 			this.$projectData.initializeProjectData(pathToProject);
-			return !!this.$projectData.projectDir && !!this.$projectData.projectId;
+			return !!this.$projectData.projectDir && !!this.$projectData.projectIdentifiers;
```

Comment:
Should check this. Not sure if we should verify for both iOS and android identifiers here or only for at least one.